src/widgets/trainers/ui/trainers-card/trainer-card.tsx

```diff
@@ -11,7 +12,7 @@ export const TrainerCard = ({ name, bio, role, photo }: TrainerProps) => {
       </div>
       <div className="right">
         <h2 className="card-title">{name}</h2>
-        <h3 className="card-subtitle">{role}</h3>
+        <Subtitle>{role}</Subtitle>
```

Comment:
I don't think we need these styles now.

![image](https://github.com/user-attachments/assets/ea22f9a1-0f9e-4ab2-b103-5e16bdd12a55)